plugin/src/main/scala/Giter8Plugin.scala

```diff
@@ -123,7 +123,7 @@ object Giter8Plugin extends sbt.AutoPlugin {
         } else Nil
 
         // copy test script or generate one
-        val script = new File(out, "test")
+        val script = new File(out, ts.getName)
```

Comment:
this is a general fix, the test script is always created as "test" while the correct name should be the one specified by `ts`.